templates/components/dropdown/limit.html.twig

```diff
@@ -40,7 +40,8 @@
 {% endif %}
 
 {% if not no_onchange %}
-   {% set href = "location.href='" ~ href ~ "glpilist_limit='+this.value+'" ~ additional_params ~ "'" %}
+   {% set href_separator = '?' in href ? '&' : '?' %}
+   {% set href = "location.href='" ~ href ~ href_separator ~ "glpilist_limit='+this.value+'" ~ additional_params ~ "'" %}
```

Comment:
There may be a theoriccal case where href contains parameters but no initial "?", no ?

Reply:
Do you have an example? If this template is going to add parameters to the URL, it should determine itself if the separator it needs to use is ? or & rather than forcing the caller template/function to add the next separator character manually.

Reply:
I'm not sure it exists, that's why I said theorical.
You may ignore